client/src/http-common.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+export default axios.create({
+  baseURL: "http://localhost:5000/",
```

Comment:
We need to figure out if we can bypass hardcoded url by using the window.location

Reply:
Yes, I tried the `window.location` and `'/'`, but that does not work.

Reply:
Okay let's keep it for now mabye we can use a env for it